src/dapp-tests/default.nix

```diff
@@ -0,0 +1,31 @@
+{ pkgs }:
+
+let
+  ds-test-src = pkgs.fetchFromGitHub {
+    owner = "dapphub";
+    repo = "ds-test";
+    rev = "eb7148d43c1ca6f9890361e2e2378364af2430ba";
```

Comment:
Very open to using niv here.

Reply:
But maybe in a separate PR, where we use it for the whole repo and not just for this library.

Reply:
I could see us using `master` here, to ensure that what's being tested is the same as people's experience using `dapp install ds-test`

Reply:
WDYM with using master? I took the last commit on master on the repo, and we have to pin one commit for this to be reproducible.. What do you have in mind?

Reply:
I had in mind not pinning it, making it non reproducible with the idea that whats being tested here is the same thing that people will be using

Reply:
You can't make it non-repro, AFAIK.